src/cheri/insns/cbld_32bit.adoc

```diff
@@ -34,6 +34,13 @@ Set `{cd}.tag=1` if:
 +
 Otherwise, set `{cd}.tag=0`
 
+#New since last ARC review#
```

Comment:
Maybe worth adding a start+end for the new since review? Or we highlight the whole region in yellow?

Reply:
I've no idea how asciidoctor would compose formatting, so I've gone with begin/end.